src/style.css

```
@@ -0,0 +1,55 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.searchBig,
```

```suggestion
.search-big,
```
it's better to name classes with kebab-case